end-to-end-tests-localdb/junit/results-0-1..xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@pvannierop same here - prolly good to exclude

Reply:
File removed.